spark/src/test/scala/org/apache/spark/sql/CometTestBase.scala

```diff
@@ -75,7 +75,6 @@ abstract class CometTestBase
     conf.set(MEMORY_OFFHEAP_SIZE.key, "2g")
     conf.set(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key, "1g")
     conf.set(SQLConf.ADAPTIVE_AUTO_BROADCASTJOIN_THRESHOLD.key, "1g")
-    conf.set(SQLConf.COALESCE_PARTITIONS_ENABLED.key, "false")
```

Comment:
We don't need to set `SQLConf.COALESCE_PARTITIONS_ENABLED.key` false now.

We also need to remove this to trigger test failure https://github.com/apache/datafusion-comet/pull/553/files#r1730694210.